feature-libs/cart/saved-cart/components/add-to-saved-cart/add-to-saved-cart.component.spec.ts

```diff
@@ -60,14 +71,13 @@ describe('AddToSavedCartComponent', () => {
           provide: SavedCartFormLaunchDialogService,
           useClass: MockSavedCartFormLaunchDialogService,
         },
+        { provide: LaunchDialogService, useClass: MockLaunchDialogService },
       ],
     }).compileComponents();
 
     isLoggedInSubject$.next(false);
     routingService = TestBed.inject(RoutingService);
-    savedCartFormLaunchDialogService = TestBed.inject(
-      SavedCartFormLaunchDialogService
-    );
+    launchDialogService = TestBed.inject(LaunchDialogService);
```

Comment:
Hey, nope. Unit tests are not part of lib - they are not exported.

Reply:
Even though unit tests are not exported, we're still supporting the deprecated code until 4.0. 

So what I was trying to say is we would support both, and have a TODO to remove the old one when we work on the deprecation ticket. Because removing the assertion that was being used with the old deprecated service will not longer be tested and we won't know if it's still functional for other 3.3.x releases.